assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -290,7 +290,11 @@
     }
 
     .form {
-        display: inline;
+        display: inline-block;
+
+        @media (min-width: $screen-xxsmall) and (max-width: $screen-medium) {
+            display: inline;
+        }
```

Comment:
Can you please check your solution for products with a lot of variant option values? 
For example 
<img width="1156" alt="many-option-values" src="https://user-images.githubusercontent.com/66325265/117766892-e97b0b80-b238-11eb-8222-142208f57814.png">